versions/3.0.md

```diff
@@ -69,17 +69,17 @@ The schema exposes two types of fields. Fixed fields, which have a declared name
 
 ### File Structure
 
-The Swagger representation of the API is made of a single file. However, parts of the definitions can be split into separate files, at the discretion of the user. This is applicable for `$ref` fields in the specification as follows from the [JSON Schema](http://json-schema.org) definitions.
+The OAS representation of the API is made of a single file. However, parts of the definitions can be split into separate files, at the discretion of the user. This is applicable for `$ref` fields in the specification as follows from the [JSON Schema](http://json-schema.org) definitions.
 
-By convention, the Swagger specification file is named `swagger.json`.
+By convention, the Open API Specification (OAS) file is named `swagger.json`.
```

Comment:
Looks like a missed space on `Open API` (reference to OAS files are `OpenAPI` everywhere else).